bootstrap.sh

```diff
@@ -49,6 +49,18 @@ program_exists() {
     local ret='0'
     type $1 >/dev/null 2>&1 || { local ret='1'; }
```

Comment:
I don't see why we shouldn't do it.

At some point we should rerun the program checkbashisms over the bootstrap script and see if we can achieve total compatibility.  That would require doing something about the BASH LINENO and FUNC stuff tho.